Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicRequestBus.h

```diff
@@ -47,9 +49,9 @@ namespace AzToolsFramework
             /**
              * Create a prefab out of the entities provided, at the path provided, and keep it in memory.
              * Automatically detects descendants of entities, and discerns between entities and child instances.
-             * Return whether the creation succeeded or not.
+             * Return the container entity id of the prefab created if creation succeeded.
```

Comment:
nit : It returns an outcome object, not just the container entity id.

Reply:
Fixed.